tools/qontract_cli.py

```diff
@@ -491,6 +491,57 @@ def service_owners_for_rds_instance(ctx, aws_account, identifier):
     print_output(ctx.obj['output'], service_owners, columns)
 
 
+@get.command()
+@click.pass_context
+def sre_checkpoints(ctx):
+    sre_checkpoints_raw = queries.get_sre_checkpoints()
+    apps = queries.get_apps()
+
+    def full_name(app):
+        name = app['name']
+        if app.get('parentApp'):
+            parent_app = app['parentApp']['name']
+            name = f"{parent_app}/{name}"
+        return name
+
+    sre_checkpoints = {}
+    for checkpoint in sre_checkpoints_raw:
```

Comment:
I'm very confused with all these `sre_checkpoints`, `checkpoint`, `sre_checkpoints_raw`... a wise man told me that naming is one of the two hardest things in computing. The others being where to put things and off-by-one errors.